### ISSUE TITLE
JSON  parse error when invalid :op specified on invoke of :application-autoscaling client

### ISSUE THREAD

**Issue by sparkofreason** [cognitect-labs/aws-api#62]
deps versions:
```clj
com.cognitect.aws/api                     {:mvn/version "0.8.266"}
com.cognitect.aws/endpoints               {:mvn/version "1.1.11.481"}
com.cognitect.aws/application-autoscaling {:mvn/version "705.2.404.0"}
```

```clj
(require '[cognitect.aws.client.api :as aws])

(def aa (aws/client {:api :application-autoscaling}))

(aws/validate-requests aa true)

(aws/invoke aa {:op :DescribeScalingPolicie
                :request {:ServiceNamespace "ecs"
                          :ResourceId "service/fib/fib-app-1da79ee"}})
```
The call to invoke returns:
```clj
{:cognitect.anomalies/category :cognitect.anomalies/fault,
 :cognitect.aws.client/throwable #error{:cause "JSON error (unexpected character): <",
                                        :via [{:type java.lang.Exception,
                                               :message "JSON error (unexpected character): <",
                                               :at [clojure.data.json$_read invokeStatic "json.clj" 226]}],
                                        :trace [[clojure.data.json$_read invokeStatic "json.clj" 226]
                                                [clojure.data.json$_read invoke "json.clj" 177]
                                                [clojure.data.json$read invokeStatic "json.clj" 272]
                                                [clojure.data.json$read doInvoke "json.clj" 228]
                                                [clojure.lang.RestFn applyTo "RestFn.java" 139]
                                                [clojure.core$apply invokeStatic "core.clj" 659]
                                                [clojure.core$apply invoke "core.clj" 652]
                                                [clojure.data.json$read_str invokeStatic "json.clj" 278]
                                                [clojure.data.json$read_str doInvoke "json.clj" 274]
                                                [clojure.lang.RestFn invoke "RestFn.java" 439]
                                                [cognitect.aws.protocols.common$json_parse_error
                                                 invokeStatic
                                                 "common.clj"
                                                 32]
                                                [cognitect.aws.protocols.common$json_parse_error invoke "common.clj" 30]
                                                [cognitect.aws.protocols.json$eval12134$fn__12137 invoke "json.clj" 48]
                                                [clojure.lang.MultiFn invoke "MultiFn.java" 238]
                                                [cognitect.aws.client$handle_http_response invokeStatic "client.clj" 46]
                                                [cognitect.aws.client$handle_http_response invoke "client.clj" 40]
                                                [cognitect.aws.client$send_request$fn__11781 invoke "client.clj" 67]
                                                [clojure.core$map$fn__5583$fn__5584 invoke "core.clj" 2734]
                                                [clojure.core.async.impl.channels$chan$fn__2034
                                                 invoke
                                                 "channels.clj"
                                                 300]
                                                [clojure.core.async.impl.channels.ManyToManyChannel
                                                 put_BANG_
                                                 "channels.clj"
                                                 83]
                                                [clojure.core.async$put_BANG_ invokeStatic "async.clj" 164]
                                                [clojure.core.async$put_BANG_ invoke "async.clj" 157]
                                                [cognitect.http_client.Client$fn$reify__10716
                                                 onComplete
                                                 "http_client.clj"
                                                 236]
                                                [org.eclipse.jetty.client.ResponseNotifier
                                                 notifyComplete
                                                 "ResponseNotifier.java"
                                                 193]
                                                [org.eclipse.jetty.client.ResponseNotifier
                                                 notifyComplete
                                                 "ResponseNotifier.java"
                                                 185]
                                                [org.eclipse.jetty.client.HttpReceiver
                                                 terminateResponse
                                                 "HttpReceiver.java"
                                                 454]
                                                [org.eclipse.jetty.client.HttpReceiver
                                                 responseSuccess
                                                 "HttpReceiver.java"
                                                 401]
                                                [org.eclipse.jetty.client.http.HttpReceiverOverHTTP
                                                 messageComplete
                                                 "HttpReceiverOverHTTP.java"
                                                 268]
                                                [org.eclipse.jetty.http.HttpParser parseContent "HttpParser.java" 1386]
                                                [org.eclipse.jetty.http.HttpParser parseNext "HttpParser.java" 1203]
                                                [org.eclipse.jetty.client.http.HttpReceiverOverHTTP
                                                 parse
                                                 "HttpReceiverOverHTTP.java"
                                                 158]
                                                [org.eclipse.jetty.client.http.HttpReceiverOverHTTP
                                                 process
                                                 "HttpReceiverOverHTTP.java"
                                                 119]
                                                [org.eclipse.jetty.client.http.HttpReceiverOverHTTP
                                                 receive
                                                 "HttpReceiverOverHTTP.java"
                                                 69]
                                                [org.eclipse.jetty.client.http.HttpChannelOverHTTP
                                                 receive
                                                 "HttpChannelOverHTTP.java"
                                                 90]
                                                [org.eclipse.jetty.client.http.HttpConnectionOverHTTP
                                                 onFillable
                                                 "HttpConnectionOverHTTP.java"
                                                 113]
                                                [org.eclipse.jetty.io.AbstractConnection$ReadCallback
                                                 succeeded
                                                 "AbstractConnection.java"
                                                 273]
                                                [org.eclipse.jetty.io.FillInterest fillable "FillInterest.java" 95]
                                                [org.eclipse.jetty.io.SelectChannelEndPoint$2
                                                 run
                                                 "SelectChannelEndPoint.java"
                                                 75]
                                                [org.eclipse.jetty.util.thread.strategy.ExecuteProduceConsume
                                                 produceAndRun
                                                 "ExecuteProduceConsume.java"
                                                 213]
                                                [org.eclipse.jetty.util.thread.strategy.ExecuteProduceConsume
                                                 run
                                                 "ExecuteProduceConsume.java"
                                                 147]
                                                [org.eclipse.jetty.util.thread.QueuedThreadPool
                                                 runJob
                                                 "QueuedThreadPool.java"
                                                 654]
                                                [org.eclipse.jetty.util.thread.QueuedThreadPool$3
                                                 run
                                                 "QueuedThreadPool.java"
                                                 572]
                                                [java.lang.Thread run "Thread.java" 748]]}}
```
Direct validation of the `:op` value and a corresponding error would be helpful here.

**Comment by dchelimsky** [comment]
Added on master, will be part of the next release.

Thanks for the suggestion!

**Comment by dchelimsky** [comment]
Released in 0.8.273
